_pkgdown.yml

```diff
@@ -9,13 +9,15 @@ template:
 reference:
 - title: Gradescope Data Functions
   desc: >
-    These functions are to read in and process the Gradescope data.
+    These functions are to read in and process data from a grading platform such as Gradescope.
   contents:
   - read_gs
   - process_gs
   - check_data_format
   - get_id_cols
   - get_assignments
+  - read_files
+  - determine_grade_source
   
```

Comment:
I am leaving read_gs as an exported function to main backwards-compatibility with any existing code. Now it is recommended to use read_files. There is no such function read_canvas. There are read_canvas_grades and read_gradescope_grades functions, which are helper functions for read_files and read_gs. Thus, I chose not to export these helper functions since users should use read_files.